docker-scripts/klaw-docker.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Read me or description on usage, what is does could help ?

Reply:
Yeah I have the usage so if execute it you can see how the script works but I agree maybe make it more explicit where some of the other documentation is? What about a section after README.md for docker that way when looking at installing it people will see there is a docker option and I can also point to the dockerhub official releases?